### PR TITLE
Implement About window design update

### LINKED ISSUE #3446: About window design update
### 💻 Environment
Platform: Windows

### 📒 Description
Update the About window view according to the new design

### PR DESCRIPTION
**Pull request by skel35**
### 📒 Description
Update the About window view according to the new design.

Note: Pressed visual states are not supported by combobox and hyperlink (#3481), everything else is pretty much as designed.

### 🕶️ Types of changes
- **Breaking change** (fix or feature that would cause existing functionality to change)

### 🤯 List of changes
- [x] Add light and dark mode color schemes
- [x] Add light/dark mode checkbox in the Debug mode
- [x] Implemented the About window design
- [x] Developed reusable styles for ComboBox, Button (primary and accent), various text styles, Hyperlink, tool window.

### 👫 Relationships
Closes #3446 

### 🔎 Review hints
Verify that About window is implemented according to the new design specified in Zeplin.
Check both Light and Dark mode.
In Debug mode there is a dark mode checkbox on the bottom left of the main window.
To check the Update button run the app in Release configuration.
To turn on Dark mode in Release configuration you can replace `Light.xaml` with `Dark.xaml` in the App.xaml file on this line: https://github.com/toggl/toggldesktop/blob/6e0081197bdeb05388445bbaa45b1bb910be4868/src/ui/windows/TogglDesktop/TogglDesktop/App.xaml#L9
OS light/dark mode detection is not implemented yet (#3449)

### CODE REVIEW

File: src/ui/windows/TogglDesktop/TogglDesktop/ui/Resources/DesignUpdate/ToolWindow.xaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:mah="http://metro.mahapps.com/winfx/xaml/controls">
+    <ResourceDictionary.MergedDictionaries>
+        <ResourceDictionary Source="Typography.xaml" />
+    </ResourceDictionary.MergedDictionaries>
+    <Style TargetType="mah:MetroWindow" x:Key="Toggl.ToolWindow">
+        <Setter Property="ResizeMode" Value="NoResize" />
+        <Setter Property="TitleCharacterCasing" Value="Normal" />
+        <Setter Property="BorderThickness" Value="0" />
+        <Setter Property="GlowBrush" Value="Black" />
```
Comment (Abdulbaqi-Alshareef) [inline src/ui/windows/TogglDesktop/TogglDesktop/ui/Resources/DesignUpdate/ToolWindow.xaml]: I think Its better to be DynamicResource and changeable with theme, so dark them has light GlowBrush and vice versa

File: src/ui/windows/TogglDesktop/TogglDesktop/ui/Resources/DesignUpdate/ToolWindow.xaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        <Setter Property="NonActiveGlowBrush">
+            <Setter.Value>
+                <SolidColorBrush Opacity="0.3" Color="Black" />
```
Comment (Abdulbaqi-Alshareef) [inline src/ui/windows/TogglDesktop/TogglDesktop/ui/Resources/DesignUpdate/ToolWindow.xaml]: same here

**Comment by skel35** [comment]
@NghiaTranUIT the icons in Zeplin were 1 pixel off. I've resized the dark mode icon and reuploaded it to match the light mode one. Please review.

File: src/ui/windows/TogglDesktop/TogglDesktop/ui/Resources/DesignUpdate/ToolWindow.xaml (hunk 1 above)
Comment (Abdulbaqi-Alshareef) [reply src/ui/windows/TogglDesktop/TogglDesktop/ui/Resources/DesignUpdate/ToolWindow.xaml]: I see and  in general both the GUI and XAML code look nice and clean.

**Pull request merged**